src/blocktypes/containers.nix

```diff
@@ -22,7 +23,38 @@
       fragment,
       fragmentRelPath,
       target,
-    }: [
+    }: let
+      inherit (n2c.packages.${currentSystem}) skopeo-nix2container;
+      img = builtins.unsafeDiscardStringContext target.imageName;
+      tags = target.meta.tags or [(builtins.unsafeDiscardStringContext target.imageTag)];
+      copyTags = let
+        skopeo = "skopeo --insecure-policy";
+        tags' =
+          builtins.toFile "${target.name}-tags.json" (builtins.unsafeDiscardStringContext (builtins.concatStringsSep "\n" tags));
+      in ''
+        export PATH=${skopeo-nix2container}/bin:$PATH
+
+        copy_tags() {
```

Comment:
Maybe `copy_tags` is a misnomer, since this effectivly loads/publishes an entire image (if need be).

Reply:
right we are copying tags to the image registry. I used the same sematic as copy_to

Reply:
Maybe just `copy` so that it can mean both? I don't think about a "payload", when I hear the word "tag", which is true for n+1, indeed. But not for n.